extensions/adobe/experience/workfront/workobject.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/workfront/object",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Work Object",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Base work related object",
+  "definitions": {
+    "wfobject": {
+      "properties": {
+        "workfront:objectName": {
+          "title": "Name",
+          "description": "The title or name associated with a given instance of the object",
+          "type": "string"
+        },
+        "workfront:parentID": {
+          "title": "Parent Object ID",
+          "description": "The @id of a parent object if it exists",
+          "type": "string"
+        },
+        "workfront:objectType": {
+          "title": "Object Type",
+          "description": "Basic string value representing the high level type of object",
+          "type": "string"
+        },
+        "workfront:lastUpdatedByID": {
```

Comment:
Should we consider using the "External Source System Audit Details" field group here?  It has the following two properties:

1. lastUpdatedBy
2. lastUpdatedDate